node/external/transactionAPI/gasUsedAndFeeProcessor.go

```diff
@@ -58,24 +58,17 @@ func (gfp *gasUsedAndFeeProcessor) computeAndAttachGasUsedAndFee(tx *transaction
 	}
 
 	hasRefundForSender := false
-	totalRefunds := big.NewInt(0)
 	for _, scr := range tx.SmartContractResults {
 		if !scr.IsRefund || scr.RcvAddr != tx.Sender {
 			continue
 		}
+		if scr.RcvAddr != tx.Sender {
```

Comment:
this conditional is dead code as it is already checked at line 62

Reply:
pushed